db/migrate/20191104001545_add_title_to_team_members.rb

```diff
@@ -0,0 +1,5 @@
+class AddTitleToTeamMembers < ActiveRecord::Migration[5.2]
```

Comment:
In the future, we should not be creating 3 different migrations to add each column. 1 migration can add as many columns as you want. This simplifies things for roll-up, deploy, and (if necessary) roll-back.

Reply:
Ok, gotcha. I added them at 3 different times so I didn't know if adding to one that was already run would cause any issues. I guess I could have done some research first.